src/Twilio.TaskRouter/Reservations.Async.cs

```diff
@@ -249,6 +261,24 @@ public virtual void ListReservationsForWorker(string workspaceSid, string worker
             if (!String.IsNullOrEmpty(redirectUrl))
                 request.AddParameter("RedirectUrl", redirectUrl);
 
+            if (!String.IsNullOrEmpty(conferenceRoomName))
+                request.AddParameter("ConferenceRoomName", conferenceRoomName);
+
+            if (!String.IsNullOrEmpty(conferenceTo))
+                request.AddParameter("ConferenceTo", conferenceTo);
+
+            if (!String.IsNullOrEmpty(conferenceFrom))
+                request.AddParameter("ConferenceFrom", conferenceFrom);
+
+            if (!String.IsNullOrEmpty(conferencePostWorkActivitySid))
+                request.AddParameter("ConferencePostWorkActivitySid", conferencePostWorkActivitySid);
+
+            if (!String.IsNullOrEmpty(conferenceTimeout))
+                request.AddParameter("ConferenceTimeout", conferenceTimeout);
+
+            if (!String.IsNullOrEmpty(conferenceStatusCallbackUrl))
```

Comment:
No unsetting. On a POST to a Reservation, the user would set instruction to be "conference" and the additional parameters specific to "Conference". Same thing as the other instructions on a Reservation (enqueue, dequeue, etc).